tests/helpers/passwordHash.test.js

```diff
@@ -0,0 +1,40 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+import bcrypt from 'bcryptjs';
+import passwordHash from '../../server/helpers/passwordHash';
+
+describe('#PasswordHash', () => {
+  const hashPasswordSpy = sinon.spy(bcrypt, 'hashSync');
```

Comment:
Nice work with your sinon @Danielshow. You would need to restore your spy so that it wont stop other test from using the spy on that function. `hashPasswordSpy.restore()` should do the trick

Reply:
ok, Done